pynitrokey/nk3/secrets_app.py

```diff
@@ -326,21 +377,32 @@ def register(
             tlv8.Entry(
                 Tag.Key.value, bytes([kind.value | algo.value, digits]) + secret
             ),
-            RawBytes([Tag.Properties.value, 0x02 if touch_button_required else 0x00]),
+            RawBytes(
+                [
+                    Tag.Properties.value,
+                    0x02
+                    if touch_button_required
+                    else 0x00 | 0x04
+                    if pin_based_encryption
+                    else 0x00,
```

Comment:
This one is potentially buggy due to the missed parenthesis, resulting in ignoring `pin_based_encryption` flag.
Fixed in https://github.com/Nitrokey/pynitrokey/pull/391